appveyor.yml

```diff
@@ -8,10 +8,31 @@ branches:
   
     - gh-pages
 
+skip_commits:
+
+  files:
+
+    - .gitattributes
+
+    - .gitignore
+
+    - README.md
+
+    - CHANGELOG.md
+
+    - LICENSE
```

Comment:
Could you condense these lines, the whitespace seems unnecessary. Same below in after_build.